todo.py

```diff
@@ -32,37 +31,33 @@ def display_todo():
 #GET Route to display one todo
 @app.route('/todo/<ObjectId:id>', methods=["GET"])
 def display_single_todo(id):
-    try:
-        todo = db.todo.find_one({"_id": id})
+    todo = db.todo.find_one({"_id": id})
+    if todo:
```

Comment:
If you handle an unwanted case first then it will be much better practice.

It should be like,

if not todo:
     return jsonify({
          message: "Id not found",
          success: False
     })

todo['_id']=str(todo['_id'])
return jsonify({
     data: todo,
     success: True
})